tools/RELEASE.md

```diff
@@ -24,163 +24,318 @@ The following guide uses `v0.11.4rc1` as version for a release candidate,
 replace as necessary with your version or remove `rc..` suffix for the
 final release.
 
+To give you an idea what this process looks like we have this fabricated
+time line:
+
+ * Time to prepare for a release.
+ * A librdkafka release candidate is available.
+ * Check if any open PRs needs merging, do that.
+ * Feature freeze - don't merge any more PRs.
+ * Create a release candidate branch.
+ * Update versions, dependencies (librdkafka), etc.
+ * Create test tag to trigger CI builds, test.
+ * Work out any release build issues.
+ * librdkafka is released, update librdkafka version.
+ * Create release candidate tag, wait for CI builds, deploy to test-PyPi.
+ * Create PR, get it reviewed.
+ * When it is time for final release, merge the PR.
+ * On master, create release tag, wait for CI builds, deploy to PyPi.
+ * Announce release.
```

Comment:
something that was missed last time was updating the API docs to the confluent website, which resulted in some issues. i don't think that's included in this set of instructions?

Reply:
Good point!